src/ate/test_programs/cp.cc

```diff
@@ -227,7 +227,8 @@ int main(int argc, char **argv) {
   // Init session with FPGA DUT and load CP provisioning firmware.
   auto dut = DutLib::Create(absl::GetFlag(FLAGS_fpga));
   dut->DutFpgaLoadBitstream(fpga_bitstream_path);
-  dut->DutLoadSramElf(openocd_path, sram_elf_path);
+  dut->DutLoadSramElf(openocd_path, sram_elf_path, /*wait_for_done=*/false,
```

Comment:
Should this `wait_for_done`?

Reply:
no because the CP binary expects UJSON payloads in / out before it completes running.